nemo/core/classes/exportable.py

```diff
@@ -192,7 +192,6 @@ def export(
                             self,
                             {"forward": tuple(input_list) + tuple(input_dict.values())},
                             strict=False,
-                            optimize=True,
```

Comment:
Why remove this line?

Reply:
Are we using `torch.jit.optimized_execution()`?
I don't think any models currently use TORCHSCRIPT. Let's leave exportable.py untouched and we'll fix it if and when errors arise in the future.